### PR TITLE
Adds widget list style

### PR DESCRIPTION
**Pull request by javierarce**
screen shot 2016-02-24 at 15 53 49

CR: @xavijam 
CC: @piensaenpixel

### CODE REVIEW

**Comment by javierarce** [comment]
Not sure what should be the description of each element, but I've chosen the name of the layer.

**Comment by xavijam** [comment]
Why have we chosen BlockList/Item as name?

File: lib/assets/javascripts/cartodb3/editor/widgets/widgets-view.js [hunk 1]
```diff
@@ -29,6 +31,7 @@ module.exports = cdb.core.View.extend({
   _addWidgetItem: function (m) {
     var view = new EditorWidgetView({
       model: m,
+      layerName: this._layerDefinitionsCollection.get(m.get('layer_id')).getName(),
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/widgets/widgets-view.js]: _OPTIONAL: I'd pass the whole layer model and then I'd decide what to render in the item view._

**Comment by javierarce** [comment]
We needed a generic name, since the same design will be used for layers. After some discussion, we chose that name.

**Comment by xavijam** [comment]
Oook! no more to add :+1:

**Comment by javierarce** [comment]
retest this please

**Pull request merged**